lib/scraper.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'httparty'
+require 'nokogiri'
+require_relative 'database'
+require_relative 'vacancy_database_handler'
+
+class Scraper
+  BASE_URL = 'https://openai.com'
+  START_URL = 'https://openai.com/careers/search'
+
+  def initialize
+    @database = DataBase.new
+    @table = CreateVacancies.new
+    @vacany_handler = VacancyDatabaseHandler.new
+  end
+
+  def call
+    @database.connect
```

Comment:
Extract this login in a separate method all extracts from the class

   @database.connect
    @table.setup_table

Reply:
Did you mean to create a method login in Scraper class and invoke it in call method?

Reply:
Do you mean to make like this? 

 def connect_to_database
    @database.connect
  end

  def setup_database_table
    @table.setup_table
  end

and just invoke it in call method. 

I'm asking because im not sure that i got you propely